doc/developer-guides/contribute_guidelines.rst

```diff
@@ -275,6 +275,12 @@ source files for the hypervisor, devicemodel, and documentation:
    agreement with the `DCO`_.  See the `Commit Guidelines`_ section
    below for specific guidelines for writing your commit messages.
 
+   All commits must be mapped to a GitHub issue for a feature or bug. Add a
+   ``Tracked-On: #issue-number`` line to your commit message together
```

Comment:
@NanlinXie can you confirm whether the capitalization matters.  Can I write this as ``tracked-on`` or ``Tracked-on`` and it will still be recognized?  (In the GitHub documentation for ``fixes:`` it says that the case does not matter and ``Fixes`` or ``fixes`` works.)